dcore/advection.py

```diff
@@ -122,3 +115,69 @@ def apply(self, x_in, x_out):
         dt = self.state.timestepping.dt
         self.solver.solve()
         x_out.assign(x_in + dt*self.dq)
+
+
+class DGAdvection(Advection):
+
+    """
+    DG 3 step SSPRK advection scheme that can be applied to a scalar 
+    or vector field
+
+    :arg state: :class:`.State` object.
+    :arg V: function space of advected field - should be DG
+    :arg continuity: optional boolean
+         if True, the advection equation is of the form:
+         D_t + (u.\nabla)D = 0
+         if False, the advection equations in of the form:
```

Comment:
s/in/is/.

Note as well you can use "latex" markup here (which will then format nicer when using sphinx).

```python
:arg continuity: optional boolean.  If ``True``, the advection equation is of the form
    :math:`D_t + (u \cdot \nabla)D = 0`.  If ``False``, use :math:`D_t + \nabla \cdot (uD)`.
```